Code/Tools/ProjectManager/Source/ProjectButtonWidget.cpp

```diff
@@ -361,9 +361,10 @@ namespace O3DE::ProjectManager
         menu->addAction(tr("Edit Project Settings..."), this, [this]() { emit EditProject(m_projectInfo.m_path); });
         menu->addAction(tr("Configure Gems..."), this, [this]() { emit EditProjectGems(m_projectInfo.m_path); });
         menu->addAction(tr("Build"), this, [this]() { emit BuildProject(m_projectInfo); });
+        menu->addAction(tr("Export"), this, [this](){ emit ExportProject(m_projectInfo);});
```

Comment:
I think `Export` is too vague. The current `Build` will build the entire project, including the editor plugins, but export is more purposeful in that it exports the game launcher. This should say 'Export Game` or `Export Launcher`.